src/Model/Actions.re

```diff
@@ -187,6 +187,7 @@ type t =
   | Modals(Feature_Modals.msg)
   // "Internal" effect action, see TitleStoreConnector
   | SetTitle(string)
+  | Maximize
```

Comment:
This should be named like an event, not like a function. The reason is that the decision of what to do should be taken in the store, not in the View. For example, how would you now implement the macOS configuration setting that minimizes on double click instead?

The View should only dispatch messages that describe what happened. The store/update function is responsible for figuring out what to actually do based on that.

I realize there are plenty of examples to the contrary here. That's old debt, and I think part of the reason they are treated as function calls, or "actions", is that the type is actually called `action`. I think it would be easier to understand if it was named `event` or `msg`. And if we got rid of all the other function-call-actions, of course.

My suggestion:
```suggestion
  | TitleDoubleClicked
```

Reply:
Gotcha that makes perfect sense. Thanks again. Will push a fix in a sec.

Reply:
Renamed in the latest commit!